lib/repl.js

```diff
@@ -844,10 +844,54 @@ REPLServer.prototype.createContext = function() {
   });
 
   addBuiltinLibsToObject(context);
+  previewResults.call(this);
 
   return context;
 };
 
+function previewResults() {
+  this.on('buffer', function(line) {
+
+    Interface.prototype.clearPreview.call(this);
+
+    // no need of preview for a multiline statement
+    if (this[kBufferedCommandSymbol] !== '')
+      return;
+
+    try {
+      sendInspectorCommand((session) => {
+        session.post('Runtime.evaluate', {
+          expression: line.toString(),
+          generatePreview: true,
+          throwOnSideEffect: true,
+          timeout: 500,
+          executionContextId: this[kContextId]
+        }, (error, result) => {
+
+          if (error) {
+            // mostly possible side effect exception
```

Comment:
Its better to expect the specific exceptions and ignore them. Is that possible?

Reply:
Also if there is an error, will the `result` object always have `result` property?

Reply:
Yes `result` will always have `result` even in case of `error. Will refactor the variable to `previewResult`, so that its better readable.

Reply:
And about the exception, it can be syntax issues, side effects etc. Not sure if I can catch them specifically. @devsnek can give some light on this.